cell_classification/augmentation_pipeline.py

```diff
@@ -90,3 +91,49 @@ def get_augmentation_pipeline(params):
         ]
     )
     return augmentation_pipeline
+
+
+def prepare_tf_aug(augmentation_pipeline):
```

Comment:
We should add tests to at least make sure these functions run. If it's easy to do we can check correctness, but making sure they take an `augmentation` object and don't error would be a good minimum